packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -883,9 +885,10 @@ describe('KeyringController', () => {
         { data: typedMsgParams, from: account },
         SignTypedDataVersion.V1,
       );
-      const recovered = recoverTypedSignatureLegacy({
-        data: typedMsgParams,
-        sig: signature as string,
+      const recovered = recoverTypedSignature({
+        data: typedMsgParams as TypedDataV1,
```

Comment:
It seems like there is no need for typecasting here since you are passing `SignTypedDataVersion.V1` as the version, is that not the case?

Reply:
removed [here](https://github.com/MetaMask/controllers/pull/914/commits/e075f5bed33d0d40f63b9386a0dce773b3bc8166)